Library/Homebrew/test/utils_spec.rb

```diff
@@ -0,0 +1,340 @@
+require "utils"
+require "tempfile"
+require "utils/shell"
+
+RSpec::Matchers.alias_matcher :have_failed, :be_failed
+
+def esc(code)
```

Comment:
This is currently declaring this method globally on `Kernel`. It should be moved inside a `describe` block so it's scoped.